src/ocamlorg_frontend/pages/community.eml

```diff
@@ -9,198 +9,343 @@ Community_layout.single_column_layout
-          <p class="font-normal text-content dark:text-dark-content mt-4">
-            The <a class="font-bold text-primary dark:text-dark-primary hover:underline" href="https://types.pl/">types.pl</a> or <a class="font-bold text-primary dark:text-dark-primary hover:underline" href="https://discuss.systems/">discuss.systems</a>
-            Mastodon homeservers are both popular choices among the OCaml community.
-          </p>
-        </div>
-        <a href="irc://irc.libera.chat/#ocaml" class="card dark:dark-card p-5 rounded-xl">
-          <%s! Icons.hashtag "h-8 w-8 mb-1 text-primary dark:text-dark-primary" %>
-          <p class="text-title dark:text-dark-title font-bold text-lg mb-1">IRC Chat</p>
-          <p class="font-normal text-content dark:text-dark-content">Chat with other members of the community at #ocaml on the Libera Chat IRC network.</p>
-        </a>
-        <div class="card dark:dark-card p-5 rounded-xl">
-          <%s! Icons.matrix "h-8 w-8 mb-1 text-primary dark:text-dark-primary" %>
-          <p class="text-title dark:text-dark-title font-bold text-lg mb-1">Matrix</p>
-          <p class="font-normal text-content dark:text-dark-content mt-4">
-            Chat with the OCaml community in the <a class="font-bold text-primary dark:text-dark-primary hover:underline" href="https://matrix.to/#/#ocaml-lang:matrix.org">#ocaml-lang Matrix chat room</a>, or join the <a class="font-bold text-primary dark:text-dark-primary hover:underline" href="https://matrix.to/#/#ocaml-space:matrix.org">#ocaml-space Matrix space</a>.
-          </p>
+  <div class="container-fluid pt-6 pb-6">
+      <p class="uppercase text-sm text-content dark:text-dark-content tracking-widest font-medium my-4">Ocaml on the internet</p>
+      <h2 class="font-bold text-2xl text-title dark:text-dark-title mb-4">Join Our Communities!</h2>
+      <p class="text-content dark:text-dark-content mt-4 mb-6">Ask and answer questions, share and discuss OCaml-related articles and posts, let people know about your projects and find collaborators</p>
+      <ul>
+        <p class="text-title dark:text-dark-title text-xl font-mono mb-2">Discussion</p>
+        <li class="flex items-center item mb-3 border-b border-separator_30 dark:border-code_window pb-2">
+            <%s! Icons.discourse "h-10 mr-12 hidden md:block text-primary dark:text-dark-primary" %>
+          <div class="flex flex-col">
+            <div class="flex items-center mb-2 md:mb-0">
+              <%s! Icons.discourse "h-10 mr-4 md:hidden block text-primary dark:text-dark-primary" %>
+              <p class="text-lg text-title dark:text-dark-title font-bold">Discuss</p>
+            </div>
+            <p class="text-content dark:text-dark-content">Discuss with the community, ask questions and shareit your work on OCaml <a href="https://discuss.ocaml.org/" class="text-primary dark:text-dark-primary hover:underline">discuss.ocaml.org/</a></p>
+          </div>
+        </li>
```

Comment:
We need to make this a component, repeating all the tailwind classes all the time is pretty bad for updating this code in the future.